Modules/Filtering/ImageFilterBase/test/itkCastImageFilterTest.cxx

```diff
@@ -400,6 +400,52 @@ TestVectorImageCast2()
 }
 
 
+bool
+TestVectorImageCast3()
+{
+  // This function casts a VectorImage<float, 2>
+  // to an Image<Vector<unsigned char, 2>, 2>
```

Comment:
I guess `unsigned char` should be `float` here, right? And `2` should be `3`?

Reply:
Fixed, copy pasting error obviously. Thanks!